docs/user-guide/src/bmo/supported_hardware.md

```diff
@@ -0,0 +1,97 @@
+# Supported hardware
+
+Metal3 supports many vendors and models of enterprise-grade hardware with
+a *BMC* ([Baseboard Management Controller][bmc]) that supports one of the
+remote management protocols described in this document. On top of that, one of
+the two boot methods must be supported:
+
+1. Network boot. Most hardware supports booting a Linux kernel and initramfs
+   via TFTP. Metal3 augments it with [iPXE][ipxe] - a higher level network boot
+   firmware with support for scripting and TCP-based protocols such as HTTP.
+
+   Booting over network relies on DHCP and thus requires a *provisioning
+   network* for isolated L2 traffic between the Metal3 control plane and the
+   machines.
+
+2. Virtual media boot. Some hardware model support directly booting an ISO 9660
```

Comment:
```suggestion
1. Virtual media boot. Some hardware model support directly booting an ISO 9660
```
Markdown numbered lists are recommended to be a list of `1.` as they will be auto numbered on rendering, removing the need of adjusting them if list length changes.